template.yaml

```diff
@@ -114,6 +152,7 @@ Resources:
     Properties:
       GroupDescription: >-
```

Comment:
All changes in this file look duplication, this file is not modified otherwise and the main branch code looks the most current one. We can discard changes from this file.

Reply:
Copied content from template.yaml from main branch into this branch to remove duplication